tests/test_lambdas/test_broker.py

```diff
@@ -0,0 +1,48 @@
+import logging
+from unittest.mock import MagicMock
+
+from pytest import LogCaptureFixture
+
+from lbz.lambdas.broker import LambdaBroker
+
+
+class TestEventBroker:
+    def test_broker_works_properly(self) -> None:
+        func_1 = MagicMock()
+        mapper = {"x": func_1}
+        event = {"op": "x", "data": {"y": 1}}
+
+        LambdaBroker(mapper, event).handle()  # type: ignore
+
+        func_1.assert_called_once_with({"y": 1})
+
+    def test_broker_responds_not_implemented_when_event_type_is_not_recognized(self) -> None:
+        func_1 = MagicMock()
+        mapper = {"x": func_1}
+        event = {"op": "y", "data": {"z": 1}}
+
+        resp = LambdaBroker(mapper, event).handle()  # type: ignore
+
+        assert resp == {
+            "error": (
+                "Lambda execution error: "
+                "NotImplementedError('No handler implemented for operation: y')"
```

Comment:
This should be rather raised, exactly as we did in `EventBroker`.

Reply:
it went a little bit different way as there are only 2 cases we might want to handle globally

Reply:
I am not sure if I can agree with this new version 🤔 Need some more time to investigate it properly.